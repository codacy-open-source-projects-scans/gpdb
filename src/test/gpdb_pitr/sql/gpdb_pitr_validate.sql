-- The DELETE was not replayed so should show all 10 rows.
SELECT * FROM gpdb_two_phase_commit_before_acquire_share_lock;

-- The DELETE should have been rebroadcasted so should show no rows.
SELECT * FROM gpdb_two_phase_commit_after_acquire_share_lock;

-- The INSERT happened after the restore point was created
-- so this table should be empty.
SELECT * FROM gpdb_two_phase_commit_after_restore_point;

-- The one-phase commit should have gone through before the restore
-- point was created so it should show up.
SELECT * FROM gpdb_one_phase_commit;
