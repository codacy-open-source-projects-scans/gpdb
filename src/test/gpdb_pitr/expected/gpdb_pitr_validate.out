-- The DELETE was not replayed so should show all 10 rows.
SELECT * FROM gpdb_two_phase_commit_before_acquire_share_lock;
 num 
-----
   2
   3
   4
   7
   8
   5
   6
   9
  10
   1
(10 rows)

-- The DELETE should have been rebroadcasted so should show no rows.
SELECT * FROM gpdb_two_phase_commit_after_acquire_share_lock;
 num 
-----
(0 rows)

-- The INSERT happened after the restore point was created
-- so this table should be empty.
SELECT * FROM gpdb_two_phase_commit_after_restore_point;
 num 
-----
(0 rows)

-- The one-phase commit should have gone through before the restore
-- point was created so it should show up.
SELECT * FROM gpdb_one_phase_commit;
 num 
-----
   1
   2
(2 rows)

